node/sc/bridge_accounts.go

```diff
@@ -111,8 +110,19 @@ func (ba *BridgeAccounts) SetChildOperatorFeePayer(feePayer common.Address) erro
 	return nil
 }
 
+//SetBridgeOperatorGasLimit changes GasLimit of parent and child operator.
+func (ba *BridgeAccounts) SetBridgeOperatorGasLimit(fee uint64) {
+	ba.pAccount.gasLimit = fee
+	ba.cAccount.gasLimit = fee
+}
+
+//GetBridgeOperatorGasLimit gets value of GasLimit of operator.
+func (ba *BridgeAccounts) GetBridgeOperatorGasLimit() uint64 {
+	return ba.pAccount.gasLimit
+}
```

Comment:
The change has an effect on both chains. I think a separate setting for each chain seems nice. Please add how do you think

Reply:
Since it is out-of-scope of this pr, let's proceed with another pr after discussing it.

Reply:
I think this consideration should be put into a top priority for the next following task.